utils/build-script-impl

```diff
@@ -2096,7 +2096,7 @@ for host in "${ALL_HOSTS[@]}"; do
                     cmake_options=(
                         "${cmake_options[@]}"
 
-                        -DSQLite3_INCLUDE_DIR:PATH=$(xcrun -sdk macosx -show-sdk-path)/usr/include
+                        -DSQLite3_ROOT:="$(xcrun -sdk macosx -show-sdk-path)/usr/include"
```

Comment:
Are you sure this is correct?  I think that that the root doesn’t take the /use/include suffix.

Reply:
My rationale to switch to `SQLite3_ROOT` was to prepend this path to the search path (as per documentation for  the documentation for [find_path](https://cmake.org/cmake/help/v3.15/command/find_path.html)) instead of forcing to find the headers at the given path -- but given your feedback I starting to see now that for macOS we may want to force finding SQLite3 at known paths (so to reduce the likelyhood of being thrown off in machines that do not align with the suggested configuration).

Reply:
Incorporating this and previous feedback in 7f4d790c169a16728ad198113bd0cdea85e01e2c